workflow-core/src/main/java/com/squareup/workflow/WorkflowUpdate.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 Square Inc.
```

Comment:
Nit: copyright year for new file should be this year.

Reply:
Fixed